src/app/users/[id]/components/ApplicationDetailModal.tsx

```diff
@@ -0,0 +1,226 @@
+              {formatDateTime(data?.programInfo.createdAt)}
+            </div>
+          </div>
+        </section>
+
+        <section className="px-3 py-3">
+          <div className="py-3 font-bold text-lf">신청자 정보</div>
+          <table>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">이름</td>
+              <td className="px-4">{data?.applierInfo.name}</td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">핸드폰 번호</td>
+              <td className="px-4">
+                {formatHyphenPhone(data?.applierInfo.phone)}
+              </td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">성별</td>
+              <td className="px-4">{data?.applierInfo.gender}</td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">생년월일</td>
+              <td className="px-4">{data?.applierInfo.birthday}</td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">주소</td>
+              <td className="px-4">
+                {data?.applierInfo.address} {data?.applierInfo.addressDetail}
+              </td>
+            </tr>
+          </table>
+        </section>
+
+        {hasAdditionalInfo() && (
+          <section className="px-3 py-3">
+            <div className="py-3 font-bold text-lf">추가 정보</div>
+            {data?.answers.map((questionAnswer) => (
+              <div>
+                <div className="text-bold text-gray-500 py-2">
+                  {questionAnswer.question.question}
+                </div>
+                <div>{questionAnswer.answer}</div>
+              </div>
+            ))}
+
+            {data?.attachmentFileIds && (
+              <div className="text-bold text-gray-500 py-2">첨부 파일</div>
+            )}
+
+            {data?.attachmentFileIds.map((attachmentFileId) => (
+              <button className="border-blue font-bold text-blue bg-indigo-50 border rounded-3xl text-[11px] mx-3 p-1">
+                {attachmentFileId} 서류
+              </button>
+            ))}
+          </section>
+        )}
+
+        <section className="px-3 py-3">
+          <div className="py-3 font-bold text-lf">신청 이력</div>
+          <table>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">상태</td>
+              <td className="px-4">
+                <Dropdown />
+              </td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">신청 일시</td>
+              <td className="px-4">{formatDateTime(data?.appliedAt)}</td>
+            </tr>
+            <tr>
+              <td className="text-bold text-gray-500 py-2">
+                {data?.status === '승인' && '승인 일시'}
+                {data?.status === '반려' && '반려 일시'}
+              </td>
+              <td className="px-4">
+                {data?.status === '승인' && data?.approvedAt}
```

Comment:
data?.status에 대한 조건문이 여러번 사용되는 걸로 보아 상위에서 변수로 선언해서 사용하면 더 좋을 것 같습니다!


```suggestion
isAccept = data?.status === '승인'
isRejected = data?.status === '반려'
isCanceled = data?.status === '취소'

// 사용하는 곳
                {isAccept && '승인 일시'}
                {isRejected && '반려 일시'}
              </td>
              <td className="px-4">
                {isAccept && data?.approvedAt}
```